package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fhi-frontend-library-web-client",
-  "version": "0.11.1",
+  "version": "3.8.0",
```

Comment:
Skip this... I think the version should be 1.0.0 when we do the "relaunch" of the FHI Designsystem...